### PR TITLE
Document KEV-related content

### LINKED ISSUE #26: Update cisagov/ncats-data-dictionary to mention KEV collection and KEV flag in tickets

### PR DESCRIPTION
**Pull request by dav3r**
#  #

## 🗣 Description ##

This PR adds KEV (Known Exploited Vulnerability)-related documentation to our data dictionary.

Note that this PR is blocked, pending the deployment of the KEV-related code changes in https://github.com/cisagov/cyhy-core/pull/68.

## 💭 Motivation and context ##

It is important to keep our documentation up-to-date.
* Resolves #26.

## 🧪 Testing ##

Tested by visually reviewing the rendered Markdown output.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

File: NCATS_Data_Dictionary.md [hunk 1]
```diff
@@ -249,6 +250,17 @@ CyHy stakeholders.
...
+### kevs Collection ###
+
+The data in this collection is derived from the
+[JSON feed](https://www.cisa.gov/sites/default/files/feeds/known_exploited_vulnerabilities.json)
+of the [CISA Known Exploited Vulnerabilities
+Catalog](https://www.cisa.gov/known-exploited-vulnerabilities-catalog).
+
+- `_id` [string]: [Common Vulnerabilities and
+  Exposures](https://cve.mitre.org/)
+  identifier for a Known Exploited Vulnerability
```
Comment (jsf9k) [inline NCATS_Data_Dictionary.md]:
```suggestion
  identifier for a Known Exploited Vulnerability (KEV)
```

**Review by jsf9k — APPROVED** [review]
I have one minor suggestion, which you can take or leave.

File: NCATS_Data_Dictionary.md (hunk 1 above)
Comment (dav3r) [reply NCATS_Data_Dictionary.md]: Excellent suggestion, thanks! 👍

File: NCATS_Data_Dictionary.md [hunk 2]
```diff
@@ -472,6 +484,7 @@ CyHy stakeholders.
     Exposures](https://cve.mitre.org/) identifier
   - `cvss_base_score` [decimal]: [CVSS v2.0 base
     score](https://nvd.nist.gov/vuln-metrics)
+  - `kev` [boolean]: Is this ticket marked as a known exploited vulnerability?
```
Comment (mcdonnnj) [inline NCATS_Data_Dictionary.md]: Just a thought for consistency with the `kevs` collection.

```suggestion
  - `kev` [boolean]: Is this ticket marked as a Known Exploited Vulnerability (KEV)?
```

**Review by mcdonnnj — APPROVED** [review]
Thanks for keeping the documentation consistent with changes! I had one suggestion for your consideration but otherwise LGTM ✔

**Pull request merged**